data.go

```diff
@@ -81,6 +81,9 @@ type NodeStatusDataMsg struct {
 	NotFound bool
 }
 
+type PingDataMsg string
+type PongDataMsg string
```

Comment:
I was actually thinking more of a struct here so we could have the option to ignore later / unknown fields, but I don't think the current scheme is flexible enough for that, and it might all be moot anyway depending on what we decide for https://github.com/livepeer/go-livepeer-basicnet/issues/33 . All that being said, this is OK right now.